Dockerfile

```diff
@@ -6,6 +6,9 @@ RUN mv "$PHP_INI_DIR/php.ini-production" "$PHP_INI_DIR/php.ini" \
 	&& apt-get --yes update && apt-get --yes install libxml2-dev \
 	&& docker-php-ext-install -j$(nproc) simplexml \
 	&& sed -ri -e 's!/var/www/html!${APACHE_DOCUMENT_ROOT}!g' /etc/apache2/sites-available/*.conf \
-	&& sed -ri -e 's!/var/www/!${APACHE_DOCUMENT_ROOT}!g' /etc/apache2/apache2.conf /etc/apache2/conf-available/*.conf
+	&& sed -ri -e 's!/var/www/!${APACHE_DOCUMENT_ROOT}!g' /etc/apache2/apache2.conf /etc/apache2/conf-available/*.conf \
+    && sed -i '/CipherString = DEFAULT@SECLEVEL=2/d' /etc/ssl/openssl.cnf \
+    && sed -i -e 's/memory_limit = 128M/memory_limit = 256M/g' $PHP_INI_DIR/php.ini
 
-COPY --chown=www-data:www-data ./ /app/
\ No newline at end of file
+COPY ./ /app/
+RUN chown -R www-data:www-data /app
```

Comment:
This line is required for modern docker versions, as the --chroot flag is not available for COPY anymore.

Reply:
Thanks for the heads up. Would you mind opening a separate PR for this change?

I already added a modified version for the security level: 1df3598a7426bedee6cfdacfcf5261da28a402b8

The memory limit is already set to a very high limit of `128M`. RSS-Bridge should operate at ~10-20M on average. If memory is an issue, there is likely a bug.

Reply:
I see. I currently don't have the time to debug this. The PR may then need to wait until then.

Reply:
No?
Also this change doubles the size of the copied in files.

Reply:
Please elaborate. When or how are the files duplicated?

Reply:
> Every command in a Dockerfile runs in a separate (intermediate) container. The results are then stored as a new image layer on top of the existing ones. Adding a file in one layer and then removing, replacing or even moving it in another layer does not remove the original file from the underlying layer. Think of layers as of Git commits — a file is preserved in the git history even after you remove it from the repo.

Source: https://medium.com/@lmakarov/the-backlash-of-chmod-chown-mv-in-your-dockerfile-f12fe08c0b55

Chown is not available on Windows and never will be for Docker on Windows.

Reply:
Good point thx.

Reply:
That is an interesting read. Thanks for the heads-up :1st_place_medal: 

Looks to me like the current solution is still supported by Docker. According to the [Docker release notes](https://docs.docker.com/engine/release-notes/) for version 18.09: `Added --chown flag support for ADD and COPY commands on Windows`

Maybe there is no need to change it after all?